src/main/java/task/Task.java

```diff
@@ -0,0 +1,53 @@
+package task;
+
+/**
+ * Represents a task with a description and a completion status.
+ */
+public abstract class Task {
+    protected final String description;
+    private boolean done;
```

Comment:
Is this the best way to name a boolean variable? Perhaps something more explicit like `isDone`?